exchanges/bybit/convert_types.go

```diff
@@ -0,0 +1,114 @@
+package bybit
+
+import (
+	"github.com/thrasher-corp/gocryptotrader/currency"
+	"github.com/thrasher-corp/gocryptotrader/encoding/json"
+	"github.com/thrasher-corp/gocryptotrader/types"
+)
+
+// Consts for WalletAccountType
```

Comment:
🐒 godoc will say they're consts, so this just stutters.
```suggestion
// WalletAccountTypes
```